tests/test_quantum_gradients.py

```diff
@@ -465,13 +467,13 @@ def d_error(p, grad_method):
             ret = 0
             for d_in, d_out in zip(in_data, out_data):
                 args = np.array([d_in, p])
-                diff = (classifier(args) - d_out)
+                diff = (classifier(*args) - d_out)
                 ret = ret + 2 * diff * classifier.jacobian(args, which=[1], method=grad_method)
             return ret
 
         y0 = error(param)
         grad = autograd.grad(error)
-        grad_auto = grad([param])
+        grad_auto = grad(param)
```

Comment:
how come list was removed here?

Reply:
This one confused me for a second, but I remember now: unlike `QNode.jacobian`, `autograd.grad()` accepts unpacked `*args`. As to why it didn't fail in caching mode, but does in caching mode - very good question that I can't answer?

Reply:
^ this change allows it to pass in both caching/no-caching modes, and conforms to the expected `grad` function call.